jsettlers.graphics/src/main/resources/jsettlers/graphics/localization/labels_pl.properties

```diff
@@ -298,8 +339,69 @@ bad-folders-version = Potrzebujesz plików graficznych gry Settlers III Złota E
 place-folders-hint = Foldery GFX i SND powinny znajdować się tutaj.
 version-build = commit: %s
 
-select-settlers-3-folder = Wskarz folder oryginalnej instalacji gry Settlers III.
+select-settlers-3-folder = Wskaż folder oryginalnej instalacji gry Settlers III.
 error-no-settlers-3-folder-selected = Nie wskazano oryginalnej instalacji gry Settlers III. Gra nie zadziała bez plików graficznych. Wyłączanie...
 error-settings-not-saveable = Nie udało się zapisać ustawień. Upewnij się, że folder gry JSettlers ma możliwość zapisu.
 
 settlers-folder-still-invalid = Wskazany folder instalacji gry Settlers III jest nieprawidłowy. W kolejnym oknie możesz wskazać poprawny folder gry Settlers III.
+enter-path = Wprowadź ścieżkę
+select-valid-settlers-3-folder = Wskaż poprawny folder gry Settlers III.
+select-settlers-3-folder-header = JSettlers - Wskaż folder gry Settlers III
+error-open-url = Błąd otwierania URL
+button-start = Rozpocznij
+i-need-help-button = Pomoc
+
+main-panel-exit-button = Wyjdź
+main-panel-new-single-player-game-button = Rozpocznij grę jednoosobową
+date.date-only = rrrr-MM-dd
+general.search = Szukaj
+
+join-game-panel-number-of-players = Liczba graczy
+join-game-panel-start-resources = Materiały początkowe
+join-game-panel-new-single-player-game-title = Nowa Gra - Jeden Gracz
+join-game-panel-new-multi-player-game-title = Nowa Gra Sieciowa
+join-game-panel-join-multi-player-game-title = Dołącz do Gry Sieciowej
```

Comment:
They are used correctly.
You can see them as title when you go to "Start Single Player Game", "New Network Game" or "Join Network Game" (and select a map).

Reply:
See this:

Game uses here **start-newmultiplayer-start** instead of **join-game-panel-new-multi-player-game-title**:
![obraz](https://user-images.githubusercontent.com/3579486/79911247-5b12cc80-8420-11ea-93d2-103da4746289.png)

and here **start-joinmultiplayer-start** instead of **join-game-panel-join-multi-player-game-title**:
![obraz](https://user-images.githubusercontent.com/3579486/79911793-42ef7d00-8421-11ea-9d23-b8fc6a3aa17e.png)

Reply:
Because this panel is the start-newmultiplayer panel and apparently all labels specific to a panel start with their panel name.

Reply:
So it means that **start-newmultiplayer-title** and **start-joinmultiplayer-title** are missing.

Reply:
No it just has another name. If it bothers you that much you could just change all occurrences of start-newmultiplayer-start to start-newmultiplayer-title (most importantly in MainMenuPanel.java)

Reply:
I thought start-newmultiplayer-start and start-joinmultiplayer-start are used for some buttons. In English file, start-joinmultiplayer-start is translated to "Join". So do you say they are not used for any buttons?

Reply:
Apparently, yes. Look at MainMenuPanel

Reply:
Ok, I may have a look tomorrow.

Reply:
But if I change that, I will create a new pull request to keep order. But this one here will need to be merged first. Are you able to apply it?

Reply:
No. Only @andreas-eberle and @tom-pratt can and both haven't shown up in months

Reply:
:/

Reply:
Naming and using the name labels is not consistent in MainMenuPanel.java.